lib/vquic/quiche.c

```diff
@@ -199,6 +203,24 @@ CURLcode Curl_quic_connect(struct connectdata *conn, curl_socket_t sockfd,
     return CURLE_OUT_OF_MEMORY;
   }
 
+/* TODO: detect UNIX nicely */
+#ifndef _WIN32
```

Comment:
How can you be sure that every non Windows system is Unix?  I would suggest using `__unix__` if possible

Reply:
I don't. Windows is in most cases the special-case, not the other way around. And I think the check should be for `WIN32` as that's what we do all over.